parts/linux/cloud-init/artifacts/cse_main.sh

```diff
@@ -233,7 +233,22 @@ if [ "${NEEDS_DOCKER_LOGIN}" = "true" ]; then
     set -x
 fi
 
-logs_to_events "AKS.CSE.installKubeletKubectlAndKubeProxy" installKubeletKubectlAndKubeProxy
+if ! semverCompare ${KUBERNETES_VERSION:-"0.0.0"} "1.32.0"; then
```

Comment:
nit: maybe move this all to a helper in cse_config.sh that we can write spec tests against?